### DISCUSSION TITLE
PPOCRLabl标注表格生成的gt.txt里面的gt的值，html代码与标注的不一样。

### DISCUSSION THREAD

**Discussion by freezehe** [PaddlePaddle/PaddleOCR#13644] [Q&A]
### Search before asking

- [X] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.

- [X] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.

- [X] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### Bug

我的表格原图如图：
![1](https://github.com/user-attachments/assets/23ef46c6-c979-4370-a2a5-a77e55b9379b)
生成的gt.txt文件
[gt.txt](https://github.com/user-attachments/files/16593648/gt.txt)
我发现gt.txt的里面的属性gt的值与我表格不一样。请问这个是什么bug？我也看了你们表格标注视频，我还特意重新排序了，还是没能生成理想的html代码

### Environment

windows cuda 环境
![image](https://github.com/user-attachments/assets/ca1d9f6d-4711-430a-abea-149554020d3a)

### Minimal Reproducible Example

{"filename": "1.jpg", "html": {"structure": {"tokens": ["<tbody>", "<tr>", "<td>", "</td>", "<td>", "</td>", "<td>", "</td>", "</tr>", "<tr>", "<td>", "</td>", "<td>", "</td>", "<td", " rowspan=\"9\"", ">", "</td>", "</tr>", "<tr>", "<td>", "</td>", "<td>", "</td>", "</tr>", "<tr>", "<td>", "</td>", "<td>", "</td>", "</tr>", "<tr>", "<td>", "</td>", "<td>", "</td>", "</tr>", "<tr>", "<td>", "</td>", "<td>", "</td>", "</tr>", "<tr>", "<td>", "</td>", "<td>", "</td>", "</tr>", "<tr>", "<td>", "</td>", "<td>", "</td>", "</tr>", "<tr>", "<td>", "</td>", "<td>", "</td>", "</tr>", "<tr>", "<td>", "</td>", "<td>", "</td>", "</tr>", "</tbody>"]}, "cells": [{"tokens": ["实", "际", "压", "力", "值", "（", "b", "a", "r", ")"], "bbox": [[84, 26], [270, 26], [270, 49], [84, 49]]}, {"tokens": ["被", "校", "仪", "器", "示", "值", "（", "b", "a", "r", "）"], "bbox": [[390, 22], [604, 22], [604, 51], [390, 51]]}, {"tokens": ["测", "量", "不", "确", "定", "度", "（", "k", "=", "2", ")"], "bbox": [[709, 26], [924, 26], [924, 49], [709, 49]]}, {"tokens": ["-", "1", ".", "0", "0", "0"], "bbox": [[149, 58], [215, 58], [215, 83], [149, 83]]}, {"tokens": ["-", "0", ".", "9", "9", "1"], "bbox": [[470, 58], [533, 58], [533, 83], [470, 83]]}, {"tokens": ["U", "r", "e", "l", "=", "0", ".", "0", "7", "%"], "bbox": [[765, 155], [879, 155], [879, 185], [765, 185]]}, {"tokens": ["0", ".", "0", "0", "0"], "bbox": [[152, 91], [211, 91], [211, 115], [152, 115]]}, {"tokens": ["0", ".", "0", "1", "1"], "bbox": [[471, 89], [531, 89], [531, 116], [471, 116]]}, {"tokens": ["1", ".", "0", "0", "0"], "bbox": [[152, 124], [211, 124], [211, 148], [152, 148]]}, {"tokens": ["1", ".", "0", "1", "0"], "bbox": [[472, 124], [532, 124], [532, 148], [472, 148]]}, {"tokens": ["2", ".", "0", "0", "0"], "bbox": [[150, 154], [211, 154], [211, 182], [150, 182]]}, {"tokens": ["2", ".", "0", "1", "1"], "bbox": [[471, 155], [532, 155], [532, 183], [471, 183]]}, {"tokens": ["3", ".", "0", "0", "0"], "bbox": [[151, 190], [210, 190], [210, 214], [151, 214]]}, {"tokens": ["3", ".", "0", "1", "2"], "bbox": [[472, 190], [532, 190], [532, 214], [472, 214]]}, {"tokens": ["4", ".", "0", "0", "0"], "bbox": [[151, 223], [210, 223], [210, 247], [151, 247]]}, {"tokens": ["4", ".", "0", "1", "2"], "bbox": [[472, 223], [532, 223], [532, 247], [472, 247]]}, {"tokens": ["5", ".", "0", "0", "0"], "bbox": [[151, 254], [213, 254], [213, 282], [151, 282]]}, {"tokens": ["5", ".", "0", "1", "2"], "bbox": [[471, 254], [533, 254], [533, 282], [471, 282]]}]}, "gt": "<html><body><table><tbody><tr><td></td><td></td><td></td></tr><tr><td></td><td>实际压力值（bar)</td><td rowspan=\"9\">被校仪器示值（bar）</td></tr><tr><td>测量不确定度（k=2)</td><td>-1.000</td></tr><tr><td>-0.991</td><td>Urel=0.07%</td></tr><tr><td>0.000</td><td>0.011</td></tr><tr><td>1.000</td><td>1.010</td></tr><tr><td>2.000</td><td>2.011</td></tr><tr><td>3.000</td><td>3.012</td></tr><tr><td>4.000</td><td>4.012</td></tr><tr><td>5.000</td><td>5.012</td></tr></tbody></table></body></html>"}

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by kerry-weic** [comment]
这个应该没关系，在[表格微调](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6.1/doc/doc_ch/table_recognition.md#11-%E6%95%B0%E6%8D%AE%E9%9B%86%E6%A0%BC%E5%BC%8F)数据结构中是没有gt这个节点

**Comment by zzz-zyq** [comment]
可能是excel标注的不对，从gt看，单元格多了整整一行。试试将PPOCRLable自动生成的excel内容ctrl A全选然后清除内容和结构，然后重新手动标注excel，再导出gt。
